scalabel/label/from_mmpkl.py

```diff
@@ -452,7 +520,7 @@ def from_mmpkl_det(
         )
 
     cfg = Config(categories=[Category(name=n, subcategories=None,isThing=None,color=None) for n in kitti_used_cats], imageSize=None, attributes=None, poseSigmas=None)
-    dataset = Dataset(frames=frames, groups=groups, config=cfg)
+    dataset = Dataset(frames=frames[:100], groups=groups, config=cfg)
```

Comment:
If we can make this an argument for argparse it would be great
```suggestion
    dataset = Dataset(frames=frames[:limit_num], groups=groups, config=cfg)

Reply:
Added and tested